### ISSUE TITLE
Weird export since 2.x

### ISSUE THREAD

**Issue by marudor** [alexgibson/notify.js#66]
Hey, 
I've updated to 2.0.2 and noticed that it exports differently.
I'm using Babel + CommonJS Module Plugin.
Since 2.0.2 I have to do the following:

``` javascript
import Notify from 'notifyjs';
if (Notify.default.needsPermission) {
  Notify.default.requestPermission();
}
```

Notice the default. - apparently it exports a Object that has a default member as default.

**Comment by alexgibson** [comment]
Yeah this is expected when the babel transpiles. You can just reference `default` when importing rather than need to define it explicitly every time.

**Comment by alexgibson** [comment]
Fwiw, the source is now writen in ES6 topic you want to import directly.

**Comment by marudor** [comment]
Actually it shouldn't.
Notifyjs has 2 defaults. It should only have 1.
Thats probably a problem with jspm though.

JSPM exports an object like this:
```javascript
{
   default: NotifyJS
}
```

ES6 expects this though:
```javascript
{
    default: NotifyJS,
    __esModule: true
}
```

If the __esModule is not set babel will wrap it into a default. So we need the __esModule property.

**Comment by alexgibson** [comment]
Hmm interesting, I will take a look into this to see if it's something doable in jspm. Thanks

**Comment by marudor** [comment]
This is what babel adds to a export
```javascript
Object.defineProperty(exports, "__esModule", {
  value: true
});
```

**Comment by alexgibson** [comment]
This may also be relevent: http://stackoverflow.com/questions/33704714/cant-require-default-export-value-in-babel-6-x

**Comment by marudor** [comment]
That applies only to "normal" requires though, not to ES6 import.
The import handles the default thing.

**Comment by alexgibson** [comment]
Yeah, right now I'm unclear if this is something that jspm can fix, but will continue looking. 

Does `import { default as Notify } from 'notifyjs` work?

**Comment by marudor** [comment]
It does. It's a workaround.

**Comment by alexgibson** [comment]
Well, apparently according to the ES6 spec, a `default` export is no different to a named one. But I agree the old way was nicer. I'll see what I can work out. Thanks for filing.

**Comment by ahutchings** [comment]
The `import { default as Notify } from 'notifyjs';` workaround does not work for me when I install notifyjs via npm. The final `Notify` reference has two levels of objects with `default` properties as @marudor mentioned.

**Comment by alexgibson** [comment]
I'm going to look at removing the dependency on jspm in the future. I may look at rollup as an option.

**Comment by ahutchings** [comment]
Sounds good. I may take a stab at the rollup build unless you get there first.

**Comment by alexgibson** [comment]
Pull requests always welcome :)

**Comment by alexgibson** [comment]
This should hopefully be fixed shortly by #73

**Comment by alexgibson** [comment]
Fixed by #73
